superset/viz.py

```diff
@@ -764,6 +764,11 @@ def query_obj(self):
         return d
 
 
+class HelloWorld(BaseViz):
```

Comment:
For some reason this appears to be necessary despite being in `useLegacyApi: false`, also the content of `buildQuery` doesn't seem used at this time

Reply:
I just lost a lot of time coming to this realisation :(

Reply:
Me too. This is a really sad limitation, if only this wasn't required, adding visualisations could be much simpler and more isolated.